res/Sectors/M1105/Drakken.txt

```diff
@@ -1,6 +1,3 @@
-# Generated by http://travellermap.com
```

Comment:
There are no Hive Federation Subsector capitals in Drakken, and there should be 12-16